docker-compose.test.yml

```diff
@@ -1,21 +1,25 @@
 services:
     static:
-        image: sylius/standard:1.11-traditional
+        build:
+            dockerfile: .docker/test/Dockerfile
+            context: .
         entrypoint: ["make", "static"]
         profiles: ["static-analyze"]
         environment:
             APP_ENV: "test_cached"
             PHP_DATE_TIMEZONE: "Europe/Warsaw"
         volumes:
-            - ./:/app:delegated
-            - ./.docker/test/php.ini:/etc/php/8.0/fpm/php.ini:delegated
-            - ./.docker/test/php.ini:/etc/php/8.0/cli/php.ini:delegated
+            - ./:/srv/sylius:delegated
+            - .docker/test/php.ini:/etc/php/8.0/fpm/php.ini:delegated
+            - .docker/test/php.ini:/usr/local/etc/php/php-cli.ini:delegated
```

Comment:
If you copied the `php.ini` file to `/usr/local/etc/php/php-cli.ini` in the `Dockerfile`, why are you mounting this file again here?